### PR TITLE
refactor: replace `format!` with `concat!` for string literals

### PR DESCRIPTION
**Pull request by Integral-Tech**
- Although `format!` macro is very powerful, it introduces overhead of memory allocation on the heap compared to `&str`. Therefore, when `format!` macro is unnecessary, we should avoid using it.

### CODE REVIEW

File: yazi-dds/src/body/hi.rs [hunk 1]
```diff
@@ -24,7 +24,9 @@ impl<'a> BodyHi<'a> {
 	}
 
 	#[inline]
-	pub fn version() -> String { format!("{} {}", env!("CARGO_PKG_VERSION"), env!("VERGEN_GIT_SHA")) }
+	pub fn version() -> String {
```
Comment (sxyazi) [inline yazi-dds/src/body/hi.rs:27]:
```suggestion
	pub fn version() -> &'static str {
```

**Comment by Integral-Tech** [comment]
@sxyazi Done :)

File: yazi-dds/src/body/hey.rs [hunk 2]
```diff
@@ -15,7 +15,7 @@ pub struct BodyHey {
 impl BodyHey {
 	#[inline]
 	pub fn owned(peers: HashMap<u64, Peer>) -> Body<'static> {
-		Self { peers, version: BodyHi::version() }.into()
+		Self { peers, version: BodyHi::version().to_string() }.into()
```
Comment (sxyazi) [inline yazi-dds/src/body/hey.rs:18]: Please also change the `version`'s type to `&'static str` or `Cow<'static, str>`

Comment (Integral-Tech) [reply yazi-dds/src/body/hey.rs:18]: @sxyazi Changing type of `version` to `&'static str` gives rise to this error:
```
error[E0521]: borrowed data escapes outside of associated function
```

**Review by sxyazi — APPROVED** [review]
Thanks!

**Pull request merged**